client/src/client-only-routes/ShowCertification.js

```diff
@@ -324,25 +324,25 @@ const ShowCertification = props => {
             </Col>
             <Col md={7} sm={12}>
               <div data-cy='issue-date' className='issue-date'>
-                Issued&nbsp;
+                {t('certification.issued')}&nbsp;
                 <strong>{format(certDate, 'MMMM d, y')}</strong>
               </div>
             </Col>
           </header>
 
           <main className='information'>
             <div className='information-container'>
-              <h3>This certifies that</h3>
+              <h3>{t('certification.certifies')}</h3>
               <h1>
                 <strong>{displayName}</strong>
               </h1>
-              <h3>has successfully completed the freeCodeCamp.org</h3>
+              <h3>{t('certification.completed')}</h3>
               <h1>
                 <strong>{certTitle}</strong>
               </h1>
               <h4>
-                Developer Certification, representing approximately{' '}
-                {completionTime} hours of coursework
+                {t('certification.developer')} {completionTime} hours of
```

Comment:
@nhcarrigan the string "hours of coursework" should be localized too. Just now I realized when I started translating.

Reply:
Nice catch! This will be resolved in an upcoming PR